tools/shared.py

```diff
@@ -2470,18 +2470,26 @@ def check_closure_compiler(args, env):
   @staticmethod
   def closure_compiler(filename, pretty=True, advanced=True, extra_closure_args=[]):
     with ToolchainProfiler.profile_block('closure_compiler'):
-      env = os.environ.copy()
-      env['PATH'] = env['PATH'] + os.pathsep + get_node_directory()
-      user_args = os.environ.get('EMCC_CLOSURE_ARGS', '')
+      def add_to_path(dirname):
+        env['PATH'] = env['PATH'] + os.pathsep + dirname
```

Comment:
I think this function would be clearer after `env = `, then the reader knows what env is already.

Reply:
good point I completely missed that.